src/containers/pages/JurisdictionAssignment/AutoSelectJurisdictions/index.tsx

```diff
@@ -133,7 +133,8 @@ export const AutoSelectView = (props: JurisdictionAssignmentViewFullProps) => {
   const { startLoading, stopLoading, loading } = useLoadingReducer(initialLoadingState);
 
   React.useEffect(() => {
-    const metadataLoadingKey = startLoading('metadata');
+    const metadataLoadingKey = 'metadata';
```

Comment:
@p-netm was the intention to move this to the constants module?

Reply:
nope. At least I did not think so. The way I see it is that the intention behind this strings is for them to be only in scope within the immediate closure blocks that they are defined in. In other words I do not expect  the string `metadata` here to be used anywhere else with the same semantic meaning. ShouldI still move them to constants file.

Reply:
@moshthepitt

Reply:
Cool, makes sense.